src/Core/Commander/Providers/WMTS_Provider.js

```diff
@@ -71,6 +71,13 @@ WMTS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer
         options.mimetype = options.mimetype || 'image/png';
         options.style = options.style || 'normal';
         options.projection = options.projection || 'EPSG:3857';
+        options.ipr.description = options.ipr.description || '';
+        options.ipr.url = options.ipr.url || '';
+        options.ipr.logo = options.ipr.logo || '';
+        options.ipr.name = options.ipr.name;
```

Comment:
So all this defensive code will be virtually identical for every provider. Why not moving it in `ApiGlobe.js:preprocessLayer`? This method has already some similar initialization code and it would avoid duplicate codes.